client/fresheyes-web/src/app/components/form/CustomComponents.tsx

```diff
@@ -39,19 +39,22 @@ export const ClickableOptions = ({
   repo,
   pull_number,
   setFormValues,
-  onClickOption,
-}: PullRequest & { onClickOption: (arg: PullRequest) => Promise<void>; setFormValues: React.Dispatch<React.SetStateAction<PullRequest>> }) => {
+  title,
+}: PullRequest & {
+  setFormValues: React.Dispatch<React.SetStateAction<PullRequest>>;
+  title: string;
+}) => {
   return (
     <button
       className={`h-full w-full dark:text-white min-h-[100px] min-w-[170px] p-3 rounded-xl md:hover:scale-105 flex flex-col items-start relative border-b border-gray-200 bg-white dark:bg-gradient-to-b from-zinc-200 backdrop-blur-2xl dark:border-neutral-800 dark:bg-zinc-800/30 dark:from-inherit lg:border lg:bg-gray-200 lg:dark:bg-zinc-800/30`}
       onClick={async () => {
         setFormValues((prev) => ({ ...prev, owner, repo, pull_number }));
-        await onClickOption({ owner, repo, pull_number });
       }}
     >
       <section className=' absolute right-3 bottom-3'>
         <p className='text-xs dark:text-white border-[1.5px] p-2 px-3 rounded-3xl font-bold'>RUN</p>
       </section>
+      <p className=' uppercase text-sm font-bold underline text-start pb-2'>{title}</p>
```

Comment:
```suggestion
      <p className='uppercase text-sm font-bold underline text-start pb-2'>{title}</p>
```
nit: there are several cases like this in the code. We can address it in another pr by setting up prettier and eslint